en-US/Design.xml

```diff
@@ -937,46 +937,63 @@ STEP 14: COMMIT <emphasis>...output omitted...</emphasis> localhost/nexus:latest
+			This section describes how to use abbreviations, acronyms, initialisms, and special characters correctly in Red&nbsp;Hat documentation.
 		</para>
 
+		 <formalpara id="first-mentions">
+			<title>First Mentions</title>
 		 <para>
 			Spell out most acronyms and initialisms before using them in text, such as "The Embedded DevKit (EDK) ...".
-          	Unless the acronym or initialism stands for a proper noun, use sentence case for the spelled out version: for example, "central processing unit (CPU)".
-          	Unless required for the audience or the topic, do not spell out well-known abbreviations, such as HTML.
+           	Unless required for the audience or the topic, do not spell out well-known abbreviations, such as HTML.
 		</para>
+		</formalpara>
 
+		 <formalpara id="cap">
+			<title>Capitalization</title>
 		 <para>
-			To form the plural of an acronym, add a trailing, lowercase "s" or "es" without an apostrophe, for example, ROMs, PINs, BIOSes.
+         	Unless the acronym or initialism stands for a proper noun, use sentence case for the spelled out version: for example, "central processing unit (CPU)".
+			Not all acronyms are capitalized (for example, "spool"); see the <citetitle>IBM Style Guide</citetitle> or another suitable reference if you are unsure.
 		</para>
+		</formalpara>
 
+		 <formalpara id="articles">
+			<title>Articles</title>
 		 <para>
-			Be sure to use correct capitalization for acronyms. Not all acronyms are capitalized (for example, "spool"); see the <citetitle>IBM Style Guide</citetitle> or another suitable reference if you are unsure.
+			When deciding which articles to use, consider pronunciation. 
+			For example, use "an <abbrev>RTS</abbrev> (real-time strategy)", because <abbrev>RTS</abbrev> is an initialism and you pronounce the first character as an "R" (är). Conversely, use "a <acronym>RAM</acronym> upgrade", because <acronym>RAM</acronym> is an acronym and you pronounce it as a word (răm).
 		</para>
-
-		 <formalpara id="initialisms">
-			<title>Initialisms</title>
-			 <para>
-				An initialism is an abbreviation that consists of the first letters of words in a phrase, syllables, or some combination thereof. Each character is pronounced separately. For example, <abbrev>FTP</abbrev> is an initialism for <systemitem class="protocol">File Transfer Protocol.</systemitem>
-			</para>
-
 		</formalpara>
+
+		 <formalpara id="plurals">
+			<title>Plurals</title>
 		 <para>
-			Consider pronunciation when using articles. See <xref linkend="acronyms" /> for more information.
+			To form the plural of an acronym, add a trailing, lowercase "s" or "es" without an apostrophe, for example, ROMs, PINs, BIOSes.
 		</para>
+		</formalpara>
+
 		 <formalpara id="special-characters">
 			<title>Special Characters</title>
```

Comment:
I'm having a little trouble understanding why special characters are addressed in the section on abbreviations, acronyms, and initialisms. That is, I see the connection around using pronunciation to determine article use,  but special characters seem like a different category from the others. Along similar lines, this section provides guidance about using special characters, but they are not defined in the previous section along with the others. To address this gap, consider adding an entry defining "Special Characters" (in the context of naming?) in the previous section.